torch/csrc/distributed/c10d/init.cpp

```diff
@@ -3400,6 +3400,11 @@ for details.
 #ifdef NCCL_HAS_NVLS_CTAS
       .def_readwrite("nvls_ctas", &ncclConfig_t::nvlsCTAs)
 #endif
+      .def_property_readonly(
+          "ptr",
```

Comment:
Do you mind naming it as `unsafe_get_ptr`? The reason is that in line 3382 above the object is created via `std::make_unique` thus this object could disappear after one gets the inner pointer. (With the rename, this entry would probably look more like a method than a property.)